packages/jupyter-ai/package.json

```diff
@@ -78,18 +78,15 @@
     "@mui/icons-material": "^5.11.0",
     "@mui/material": "^5.11.0",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0",
-    "react-markdown": "^8.0.6",
-    "react-syntax-highlighter": "^15.5.0",
-    "rehype-katex": "^6.0.2",
-    "remark-math": "^5.1.1"
+    "react-dom": "^18.2.0"
   },
   "devDependencies": {
     "@babel/core": "^7.0.0",
     "@babel/preset-env": "^7.0.0",
     "@jupyterlab/builder": "^4",
     "@jupyterlab/testutils": "^4",
     "@types/jest": "^29",
+    "@types/react-dom": "^18.2.18",
     "@types/react-syntax-highlighter": "^15.5.6",
```

Comment:
Last suggestion, I promise - sorry for not catching this earlier, I only saw this after reading bdde01b39082fadf075a724ec2d89a9ac5f38e43 commit message.

```suggestion
```

Reply:
@krassowski that's a good one, missed it. No worries, really appreciate the comments, please keep it coming if there is anything to add.